tachyon/zk/plonk/permutation/permutation_assembly_unittest.cc

```diff
@@ -0,0 +1,141 @@
+#include "tachyon/zk/plonk/permutation/permutation_assembly.h"
+
+#include <iostream>
+
+#include "gtest/gtest.h"
+
+#include "tachyon/base/buffer/vector_buffer.h"
+#include "tachyon/math/elliptic_curves/bn/bn254/g1.h"
+#include "tachyon/math/elliptic_curves/short_weierstrass/test/curve_config.h"
+#include "tachyon/math/polynomials/univariate/univariate_evaluation_domain.h"
+#include "tachyon/math/polynomials/univariate/univariate_evaluation_domain_factory.h"
+#include "tachyon/zk/plonk/permutation/permutation_argument.h"
+#include "tachyon/zk/plonk/permutation/permutation_keys.h"
+
+namespace tachyon::zk {
+namespace {
+
+constexpr static size_t MaxDegree = 5;
+constexpr static size_t N = MaxDegree + 1;
+
+using Curve = math::bn254::G1Curve;
+using F = typename Curve::ScalarField;
+using B = typename F::BigIntTy;
+
+using ArgTy = PermutationArgument<Curve, MaxDegree>;
+using AsmTy = PermutationAssembly<Curve, MaxDegree>;
+using AsmMapping = typename AsmTy::Mapping;
+using AsmLabel = typename AsmTy::Label;
+using EvalsPolyTy = typename PermutationAssembly<Curve, MaxDegree>::EvalsPolyTy;
+
+std::vector<AnyColumn> COLUMNS = {AnyColumn(0), AdviceColumn(1), FixedColumn(2),
+                                  InstanceColumn(3)};
```

Comment:
Do not declare `std::vector<>` global variable.

Seee https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables